src/Domain/CoordinateMaps/CoordinateMap.tpp

```diff
@@ -165,6 +165,60 @@ CoordinateMap<SourceFrame, TargetFrame, Maps...>::inverse_impl(
              : boost::optional<tnsr::I<T, dim, SourceFrame>>{};
 }
 
+namespace detail {
+template <typename T, typename Map, size_t Dim,
+          Requires<not domain::is_jacobian_time_dependent_v<Map, T>> = nullptr>
```

Comment:
Given that this is in a detail namespace, is it better to transmit the boolean information of whether it is time dependent as a function overload rather than SFINAE? I recall previous code-reviews preferring overloads to SFINAE for marginally improved compile times.

Reply:
Agreed, because of the conflicts that arose with a fixup I decided to squash right away because I wanted to have you look at the exact changes being proposed :)